GameServer/Logic/GameObjects/Inhibitor.cs

```diff
@@ -97,6 +97,11 @@ public override void update(long diff)
             base.update(diff);
         }
 
+        public bool GetRespawnAnnounced()
```

Comment:
This should be a C# property (`public bool RespawnAnnounced { get; private set; }`)